operational_analysis/methods/turbine_long_term_gross_energy.py

```diff
@@ -77,30 +81,53 @@ def __init__(self, plant, max_power_filter = 0.85, wind_bin_thresh=2):
         # Set number of 'valid' counts required when summing data to daily values
         self._num_valid_daily = 60. / self._time_conversion[self._plant._scada_freq] * 24
         
+
+        
         # Initially sort the different turbine data into dictionary entries
+        logger.info("Processing SCADA data into dictionaries by turbine (this can take a while)")
         self.sort_scada_by_turbine()
         
 
     @logged_method_call
-    def run(self, reanal_subset):
+    def run(self, reanal_subset = ['erai', 'ncep2', 'merra2'], max_power_filter = 0.85, 
```

Comment:
Add more parameters to the run() method, including plotting indicators and save file